srcts/src/imageutils/initCoordmap.ts

```diff
@@ -50,10 +50,14 @@ function findDims($el: JQuery<HTMLElement>) {
   // If there's any padding/border, we need to find the ratio of the actual
   // element content compared to the element plus padding and border.
   const contentRatio = {
-    x: $el.width() / $el.outerWidth(),
-    y: $el.height() / $el.outerHeight(),
+    x: ($el.width() ?? NaN) / ($el.outerWidth() ?? NaN),
+    y: ($el.height() ?? NaN) / ($el.outerHeight() ?? NaN),
   };
 
+  // If any number couldn't be found, replace NaN with 1.
+  if (Number.isNaN(contentRatio.x)) contentRatio.x = 1;
+  if (Number.isNaN(contentRatio.y)) contentRatio.y = 1;
```

Comment:
It is possible that values are undefined. Safe guard with 1

Reply:
For this whole file, let's use `!` instead--I'd rather just make the compiler happy than to change behavior for code that we think is working